MathLib/LinAlg/Lis/LisMatrix.cpp

```diff
@@ -87,6 +87,7 @@ void LisMatrix::setZero()
 
 int LisMatrix::setValue(IndexType rowId, IndexType colId, double v)
 {
+    if (v == 0.0) return 0;
```

Comment:
How does this influence the generation of the sparsitty pattern of the LisMatrix? I assume there will be no entry in the sparsitty pattern. Will/Can this be a problem when the matrix is reused in the next assembly iteration?
Same question for the handling in `LisMatrix::add()`.